### ISSUE TITLE
fyi: if any of these efts appeal, please take 'em,

### ISSUE THREAD

**Issue by J-Sarnoff** [dsiem/ErrorFreeTransforms.jl#1]
https://github.com/J-Sarnoff/ErrorFreeArith.jl

**Comment by dsiem** [comment]
Looks good. However, this project is essentially abandoned as currently I have no time to work on it. It was meant to be part of another project that I haven't even started yet. There is also https://github.com/simonbyrne/DoubleDouble.jl which seems to be abandoned as well. I hope to come back to working on this project but for the moment I can only suggest to create a package that supersedes mine.

**Comment by J-Sarnoff** [comment]
ok -- do you mind if keep the name?

**Comment by dsiem** [comment]
Not really, but I have also no idea how exactly the Julia package infrastructure works.

**Comment by J-Sarnoff** [comment]
nor do I, so I will name it something else (sometime)
